apps/site/public/demo-utils/touchai-lite-math.js

```
@@ -0,0 +1,253 @@
+    }
+
+    function renderFormula(source, displayMode) {
+        if (window.katex && typeof window.katex.renderToString === 'function') {
+            try {
+                return window.katex.renderToString(String(source), {
+                    displayMode,
+                    throwOnError: false,
+                    strict: 'ignore',
+                    trust: false,
+                });
+            } catch (error) {}
```

ESLint

[error] 'error' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Empty block statement.

(no-empty)